qa/L1_jupyter_notebooks_TU102/setup_optflow.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash -e
+
+optflow_so=`find /usr -name libnvidia-opticalflow.so`
+optflow_so_1=`find /usr -name libnvidia-opticalflow.so.1`
+if [ -z $optflow_so ]; then
+    if [ ! -z $optflow_so_1 ]; then
+        echo "INFO: Could not find libnvidia-opticalflow.so"
+        echo "INFO: libnvidia-opticalflow.so.1 found: $optflow_so_1"
+        echo "IFNO: Creating a symlink $optflow_so_1 => $optflow_so"
+        optflow_so=`dirname $optflow_so_1`/libnvidia-opticalflow.so
+        ln -s $optflow_so_1 $optflow_so
```

Comment:
Shouldn't we just be running `ldconfig` instead of manually making those symlinks?

Note that either way this only works because we run the test containers as root.  Probably not much we can do about that (fortunately no instant plans to change this, either :) ).

Reply:
Although I guess for nvidia-docker1 the path this library would appear in would be `/usr/local/nvidia/lib*` which probably ldconfig might not see, depending on how the ld.so.conf is done for that image.  :hmm:

Reply:
ldconfig created `libnvidia-opticalflow.so.1` but not `libnvidia-opticalflow.so` :(